### ISSUE TITLE
Unbrick Guide

### ISSUE THREAD

**Issue by Committee** [Plailect/Guide_3DS#1278]
Possible to add a section on how to unbrick using NTRBoot and Godmode9?

**Comment by Plailect** [comment]
It's possible. I'll look into it.

**Comment by pbanj** [comment]
are you asking for a guide for unbricking with or without a nand backup? if with, you don't really need a guide. just load godmode9 and restore the nand backup.

**Comment by Committee** [comment]
For both, since the guide is the bible for 3DS works I think a guide on how to unbrick would be beneficial to users as there would be step by step instructions on how to do so.

**Comment by weirdal3333** [comment]
There are different kinds of bricks. 
Examples:
1. Blue screen
2. NAND brick
3. Black screen
Etc.

There should be different guides for different bricks.
OR
There should be one guide that fixes everything (not data related), and a different guide that deals with the NAND restore.

It's easy to unbrick a device with NTRBOOT, so maybe you could put a link in that part of the guide too?

**Comment by pokecraft98** [comment]
https://github.com/AnalogMan151/Lazarus3DS Since this exist it should help with the unbrick guide

**Comment by Plailect** [comment]
This is a really interesting project. @AnalogMan151 will have to comment as to whether they believe it is ready for use or not.

**Comment by lanleargas** [comment]
I am new to custom firmware on New 3ds. Last week I followed the 3ds guide and everthing worked, but I must have done something wrong afterwards. When I start my N3ds I get the message Failed to mount CTRNAND. I have boot9strap, Luma3DS v8.1.1 and Godmode v1.4.2
In Godmode I see in root 0: SDCARD 
2: SYSNAND TWLN
 S: SYSNAND VIRTUAL
9: RAMDRIVE
C; GAMECART
M; MEMORY VIRTUAL
X; NAND XORPADS

I have a backup and tried to restore it, but the message still comes. What should I do?

If this is not the place to ask this question, I do apologize.

**Comment by Derek1199** [comment]
One error the guide has though is that it says your need a hardmod to fix the bootrom 8046 error. This can be fixed using ntrboot. Booting GodMode9. For example: Both my firm partitions were corrupted and I used GodMode9 to unbrick. I also corrupted my nand header on purpose by editing random bytes and then used GodMode9 to fix it. It's easy to use ntrboot, and it ntrboot can fix any brick except for severe HW damage and MCU bricks.

**Comment by lanleargas** [comment]
Can I find a guide somewhere how to do this with ntrboot. Where can I download ntrboot. I think I mist have it somewhere, but no idea how it is called.

**Comment by Derek1199** [comment]
You need a NTRBoot compatible flashcart. Then you need to flash to that cart using another 3ds. You can flash GodMode9, or B9S to the cart.

**Comment by Derek1199** [comment]
@lanleargas do you get a blue bootrom error code screen on boot though? Also check which files you can see inside sysnand virtual if you don't get that blue bootrom error code screen.

**Comment by lanleargas** [comment]
I do not have a blue bootrom error on boot. I can start Luma and Godemode, but nothing else. When I want to boot the N3ds I see a red line stating: An error has occurred: and two white lines saying: Failed to mount CTRNAND. Press any button to shutdown.

In SYSNAND VIRTUAL it says
agbsave.bin
Bonus.bin
Ctrnand_full.bin  
Essential.exefs
Firm0.bin
Firm1.bin
Nand.bin
Nand_hdr.bin
Nand_minsize.bin
Sector0x96.bin
Twlmbr.bin
Twln.bin
Twlp.bin

**Comment by Derek1199** [comment]
Try pressing the a button on nand.bin and seeing if there's an option there for fixing anything. Or try pressing r+a and fixing the CMAC's for each visible drive you can see.

**Comment by lanleargas** [comment]
I can press a on nand.bin and then I can choose NAND image options and three more options. When I press NAND image options, I get three more:
mount image to drive
UPdate embedded backup
Verify file

**Comment by Derek1199** [comment]
Oh. Hmm. Try checking memory virtual drive then. Do you see boot9.bin there?

**Comment by lanleargas** [comment]
Yes, it is there.

**Comment by Derek1199** [comment]
Okay. Then go to sysnand virtual and press a on nand_hdr.bin and select copy to 0:/gm/out. Then after that press a on it and select rebuild NCSD header. Then reboot and tell me if this fixes your problem.

**Comment by lanleargas** [comment]
I copied nand_hdr.bin to 0:/gm/out. I went to that location and pressed nand_hdr.bin and then I could choose between

Show in Hexeditor
Calculate SHA-256

But I do not see the option that I can rebuild NCSD header

**Comment by Derek1199** [comment]
This sounds like hardware damage then.

**Comment by lanleargas** [comment]
In the same folder I have a two files called 170929_QEF10280880_sysnand_000.bin and one with the same name with sha. Those two files are th dump I made in case I get a brick.
Could I use Mount image to drive?

I tried to restore sysNAND a few days ago, but that did not solve the problem.

**Comment by Derek1199** [comment]
Mount image to drive just shows you the files/folders inside that partition, and mounted images are always read only. Anyway, this sounds like hardware damage.

**Comment by lanleargas** [comment]
When I try to mount the image, it says: Mounting image: failed.
When I try to restore safe I get  an error: MBR partition1 is corrupt. And then NAND dump is corrupt. I can still continue but then I will overwrite the sysNAND with the provided dump. It will leave the B9S/A9LH intact. I have B9S, boot9strap.

**Comment by lanleargas** [comment]
Hm, HW damage? So I have a useless n3ds. That really hurts.

**Comment by Derek1199** [comment]
You may have HW damage. The best suggestion I have now is try doing a CTRNand transfer.

**Comment by pokecraft98** [comment]
github issues is the not the place to get help. you can get help in the nintendo homebrew server https://discord.gg/C29hYvh

**Comment by lanleargas** [comment]
I will try to do that tomorrow.

**Comment by ForcedToRock** [comment]
Something of a bump, in regards to Lazarus 3DS. I was able to unbrick successfully, though having to rename some files for the script to be able to read them was a hurdle, as well as finding the sighax_hdr.bin file. Other than that, unbricked...to a 002-0121 error code, banning access to all online services including e-shop. I had to use an LCFS_b from a arely used 2DS. I talked with someone who used the script and they also got that error code.
